indicators.py

```diff
@@ -0,0 +1,41 @@
+import urllib.request
+import json
+
+apikey = "R1OZ339ORWVQ05E9"
+
+class Indicators: 
+
+    def __init__(self,ticker,date):
```

Comment:
maybe instantiate with a `Stock` object instead of just a string

Reply:
actually, maybe this should all be methods on the Stock object

Reply:
an Indicator object doesn't make a lot of sense

future design: a `Strategy` parent class with inheriting child classes. Each strategy looks at a `Universe` of `Stock`s and makes predictions

Reply:
The point is, `Strategy` should only need to look at `Stock` to get all technical indicators; anything else is overcomplicated.